op-chain-ops/genesis/config.go

```diff
@@ -522,6 +527,8 @@ type L1Deployments struct {
 	ProxyAdmin                        common.Address `json:"ProxyAdmin"`
 	SystemConfig                      common.Address `json:"SystemConfig"`
 	SystemConfigProxy                 common.Address `json:"SystemConfigProxy"`
+	ProtocolVersions                  common.Address `json:"ProtocolVersions"`
```

Comment:
In the previous PR, I did add a useful comment to the godoc for this struct

Reply:
Was it this exact struct? Let me try to find it, I may have missed it when diffing and cherry-picking changes from the PR

Reply:
Can't find it in the old PR, I believe it's the deploy-config struct only that had comments, the L1Deployments one does not. Feel free to suggest some (the other addresses do not have comments here either though).